x-pack/plugins/security_solution/public/overview/components/detection_response/translations.ts

```diff
@@ -68,3 +84,63 @@ export const OPEN_ALL_ALERTS_BUTTON = i18n.translate(
     defaultMessage: 'View all open alerts',
   }
 );
+
+export const VIEW_ALL_USER_ALERTS = i18n.translate(
+  'xpack.securitySolution.detectionResponse.viewAllUserAlerts',
+  {
+    defaultMessage: 'View all other user alerts',
+  }
+);
+
+export const VIEW_ALL_HOST_ALERTS = i18n.translate(
+  'xpack.securitySolution.detectionResponse.viewAllHostAlerts',
+  {
+    defaultMessage: 'View all other host alerts',
+  }
+);
+
+export const SEVERITY_CRITICAL_COLUMN = i18n.translate(
```

Comment:
Severity should exist in the file already, please pull the main branch and check again

Reply:
question:

Even though it exists, does it make sense to have multiple versions since the tables are independent? For eg - what if we decide to change the column name on one, we could accidentally change it for the other as well.

Similarly you have ALERTS_TITLE -> Alerts

but I have ALERTS_COLUMN -> Alerts

should i remove mine and use yours? 

They are used in two different ways 🤔

Reply:
resolved in slack conversation: came to the agreement to change alerts as well. the translations are being used in the same place, but if either of them become independent, thats when we'll have multiple translations